src/Tribe/Main.php

```diff
@@ -139,6 +149,44 @@ public function load_assets() {
 				),
 			)
 		);
+
+		tribe_asset(
+			$this,
+			'tribe-common',
+			'tribe-common.js',
+			array( 'tribe-clipboard' ),
+			'admin_enqueue_scripts',
+			array(
+				'localize' => array(
+					'name' => 'tribe_l10n_datatables',
+					'data' => array(
+						'aria' => array(
+							'sort_ascending' => __( ': activate to sort column ascending', 'the-events-calendar' ),
```

Comment:
Here and the next few lines: should we switch text domain (as this code is in tribe-common rather than the-events-calendar)?